src/cultionet/model.py

```diff
@@ -488,24 +493,25 @@ def fit(
     if auto_lr_find:
         trainer.tune(model=lit_model, datamodule=data_module)
     else:
-        trainer.fit(
-            model=lit_model,
-            datamodule=data_module,
-            ckpt_path=ckpt_file if ckpt_file.is_file() else None
-        )
-        # Calibrate the logits
-        temperature_model = TemperatureScaling(
-            cultionet_model=CultioLitModel.load_from_checkpoint(
-                checkpoint_path=str(ckpt_file)
-            ),
-            edge_class=edge_class,
-            class_counts=class_counts
-        )
-        temperature_trainer.fit(
-            model=temperature_model,
-            datamodule=temperature_data_module,
-            ckpt_path=temperature_ckpt_file if temperature_ckpt_file.is_file() else None
-        )
+        if not skip_train:
+            trainer.fit(
+                model=lit_model,
+                datamodule=data_module,
+                ckpt_path=ckpt_file if ckpt_file.is_file() else None
+            )
+        if refine_and_calibrate:
+            # Calibrate the logits
+            temperature_model = TemperatureScaling(
+                num_classes=num_classes,
+                edge_class=edge_class,
+                class_counts=class_counts,
+                cultionet_ckpt=ckpt_file
+            )
+            temperature_trainer.fit(
+                model=temperature_model,
+                datamodule=temperature_data_module,
+                ckpt_path=temperature_ckpt_file if temperature_ckpt_file.is_file() else None
```

Comment:
I see the temperature_ckpt_file is defined as `temperature_ckpt_file = ckpt_file.parent / 'temperature' / ckpt_file.name`
Let's say I did not temperature folder, the ckpt_path will be None and it will just use the last.ckpt?

Reply:
A temperature folder will be created if it doesn't exist. And the temperature checkpoint file will be created if `refine_and_calibrate=True`. Is that what you're asking?

Reply:
ok, thanks for clarifying. I was just wondering what happened when I do have have a temperature folder save.